pkg/shp/flags/core_envvar_value.go

```diff
@@ -0,0 +1,61 @@
+package flags
```

Comment:
Copyright header missing, though I just notice that many existing files also do not contain one.

Reply:
I think we still need to bring some tools from the build-controller to this repository as well 🤔

Reply:
Yep, opened [Ensure files have copyright headers #73](https://github.com/shipwright-io/cli/issues/73).